resources/config-defaults.edn

```diff
@@ -325,4 +325,8 @@
  :accessibility-report false ; should the axe accessibility tooling be included?
  :enable-ega false ; should the EGA features be shown in the UI
  :enable-doi false ; should the DOI features be shown in the UI
- :enable-duo false} ; should the DUO features be shown in the UI
+ :enable-duo false ; should the DUO features be shown in the UI
+
+ :enable-catalogue-table true ; show catalogue page table of items
+ :enable-catalogue-tree false ; show catalogue page tree of items
+ :catalogue-tree-show-matching-parents true} ; should parent categories be shown if children match?
```

Comment:
I wanted it to be clear whether you want to include the catalogue tree or not on the catalogue page (identical to the table). The extra option of filter showing the matching parents is by default true so I think it will be a rare option to disable it.